Sources/TSCUtility/Netrc.swift

```diff
@@ -13,15 +13,6 @@ extension AuthorizationProviding {
     }
 }
 
-#if os(Windows)
-// FIXME: - add support for Windows when regex function available
-#endif
-
-#if os(Linux)
-// FIXME: - add support for Linux when regex function available
-#endif
```

Comment:
afaict this is available since apple/swift-corelibs-foundation#1522, tests on linux for 5.2, 5.3, 5.4